### DISCUSSION TITLE
Decimal Value of Floating-Point

### DISCUSSION THREAD

**Discussion by baierd** [bitwuzla/bitwuzla#76] [Q&A]
Hi,

is it possible to recieve the decimal value of a FP?
I tried bitwuzla_term_to_string_fmt(fp_term, 10) and bitwuzla_term_value_get_str_fmt(term, 10) on FP values (created w bitwuzla_mk_fp_from_real() in my tests).

Thank you for your help!

Best,
Daniel

**Comment by aniemetz** [comment]
Bitwuzla currently does not support this. For easier tracking of this feature request, see #104.

**Reply by aniemetz** [reply]
@baierd This feature is now on main as of https://github.com/bitwuzla/bitwuzla/commit/9f7a6a3a47000c0e4262b6ac24658c3aa2e58f71. Let us know if you run into unexpected issues.
